pom.xml

```diff
@@ -170,7 +170,7 @@
     <!-- for now, not running scalafmt as part of default verify pipeline -->
     <scalafmt.skip>true</scalafmt.skip>
     <codehaus.jackson.version>1.9.13</codehaus.jackson.version>
-    <fasterxml.jackson.version>2.13.3</fasterxml.jackson.version>
+    <fasterxml.jackson.version>2.13.4</fasterxml.jackson.version>
     <fasterxml.jackson.databind.version>2.13.3</fasterxml.jackson.databind.version>
```

Comment:
Could you also update `fasterxml.jackson.databind.version` to 2.13.4?

Reply:
I forgot the reason for adding `fasterxml.jackson.databind.version`, but do we still need two version properties related to `jackson`?

Reply:
Agree with @srowen

Reply:
No, as @srowen said, we should keep it